tesseract-server/src/handlers/aggregate.rs

```diff
@@ -240,3 +265,21 @@ impl TryFrom<AggregateQueryOpt> for TsQuery {
         })
     }
 }
+
+use failure::{bail, format_err};
+use tesseract_core::names::Cut;
+use crate::logic_layer::CubeCache;
+
+fn validate_members(cuts: &[Cut], cube_cache: &CubeCache) -> Result<(), Error> {
```

Comment:
Should we move this somewhere where to logic layer could use it as well?

Reply:
Agreed! What do you think about having it in `handlers/util.rs`? (see latest push)

Reply:
That looks good to me!